docs/conduit/Development.md

```diff
@@ -0,0 +1,44 @@
+# Creating A Plugin
```

Comment:
Do we want to talk about what the "lifecycle" of a plugin is and where each function is expected to call?

Reply:
Updated. This page probably should be updated to have a separate section for each interface. For now I think the basics are covered.